src/main.rs

```diff
@@ -1,3 +1,10 @@
+// #![warn(clippy::pedantic)]
+// #![allow(clippy::uninlined_format_args)]
+// #![allow(clippy::enum_glob_use)]
+// #![allow(clippy::module_name_repetitions)]
+// #![allow(clippy::doc_markdown)]
+// #![allow(clippy::if_not_else)]
+// #![allow(clippy::ignored_unit_patterns)]
```

Comment:
I like to use this pattern in some of my projects. I find pedantic is often a bad mix of false positives, but having this here does two things:

a) quick-and-easy way to surface nit-picks, hiding classes of nits with fine control
b) documents where the project disagrees with the `pedantic` linter

Reply:
Yeah, these seem sane to me.